app/views/layouts/_footer.html.erb

```diff
@@ -1,29 +1,60 @@
+<style>
+    .footer-nav-link::after {
+        content: '';
+        position: absolute;
+        top: 0;
+        left: 0;
+        bottom: 0;
+        right: 0;
+    }
+    .nav-bar-entry {
+      position: relative;
+    }
+</style>
+
 <% if user_signed_in? %>
   <div id="navigation">
     <ul class="nav nav-pills nav-justified">
       <li class="nav-item">
-        <a class="nav-link <%= active_link_to_class(root_path, active: :exclusive)%>" href="<%= root_url %>">
-          <i class="fa fa-home"></i>
-        </a></li>
+         <label class="nav-bar-entry">
+          <!-- new path to this when dashboard is rewritten -->
+            <a class="nav-link footer-nav-link <%= active_link_to_class(root_path, active: :exclusive)%>" href="<%= root_url %>">
+            <i class="fa fa-home"></i>
+          </a>
+        </label>
+      </li>
       <li class="nav-item">
-        <a class="nav-link <%= active_link_to_class(chat_path, active: :exclusive)%>" href="<%= chat_path %>">
-          <i class="fa fa-comments"></i>
-        </a></li>
+        <!-- please update the path when reworked the chat -->
+          <label class="nav-bar-entry">
+          <a class="nav-link footer-nav-link <%= active_link_to_class(root_path, active: :exclusive)%>" href="<%= chat_path %>">
+            <i class="fa fa-comments"></i>
+          </a>
+        </label>
+      </li>
       <li class="nav-item">
-        <a class="nav-link <%= active_link_to_class(user_contacts_path(current_user), active: :exclusive)%>"
-           href="<%= user_contacts_path(current_user) %>">
-          <i class="fa fa-address-book"></i>
-        </a></li>
+        <label class="nav-bar-entry">
+          <a class="nav-link footer-nav-link <%= active_link_to_class(user_contacts_path(current_user), active: :exclusive)%>" 
+            href="<%= user_contacts_path(current_user) %>">
+            <i class="fa fa-address-book"></i>
+          </a>
+        </label>
+      </li>
       <li class="nav-item">
         <!-- please add the right path here when history view created -->
-        <a class="nav-link" href="">
-          <i class="fa fa-history"></i>
-        </a></li>
+        <label class="nav-bar-entry">
+          <a class="nav-link footer-nav-link" href="">
+            <i class="fa fa-history"></i>
+          </a>
+        </label>
+      </li>
       <li class="nav-item">
-        <a class="nav-link <%= active_link_to_class(edit_user_path(current_user), active: :exclusive)%>"
-           href="<%= edit_user_path(current_user) %>">
-          <i class="fa fa-cog"></i>
-        </a></li>
+        <label class="nav-bar-entry">
+          <a class="nav-link footer-nav-link <%= active_link_to_class(edit_user_path(current_user), active: :exclusive)%>" 
+            href="<%= edit_user_path(current_user) %>">
+            <i class="fa fa-cog"></i>
+          </a>
+        </label>
```

Comment:
What is the reason for the changes in this file? It does not seem to be related to the topic of the PR. You could remove this here/ move it to another PR to reduce complexity

Reply:
Maybe the following screenshot describes it the best: 

<img width="1220" alt="Screenshot 2021-02-05 at 17 09 16" src="https://user-images.githubusercontent.com/32880898/107066455-4b614a80-67de-11eb-812a-1710c19e730d.png">

The link of the setting page is covering the whole page.

Reply:
To solve this problem I simply wrap the link inside a lable.

Reply:
Sorry I don't have access to a PC right now. Can you reproduce this @felixauringer?

Reply:
@frcroth is it okay if I leave it in this PR? I know it's not the best solution, but at the moment I prefer the fastest one.

Reply:
![image](https://user-images.githubusercontent.com/48409110/107075179-0cd18d00-67ea-11eb-9377-ca183220cd61.png)

I'm on your branch @Alexander-Dubrawski but I resetted `app/views/layouts/_footer.html.erb` to its state on the `dev` branch and everything seems to be fine without your changes in this file. I'm not quite sure what the problem is :sweat_smile:

Reply:
That's confusing. I have the same problem on dev.  But I will double-check.

Reply:
Well..., somehow it got magically fixed. I can't reproduce it anymore either. Maybe one of the last merges fixed it. Its getting hard to keep track of.